src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="content">
+      <div class="calendar calendar--start-day_thu calendar--month-length_30">
+        <div class="day"></div>
+        <div class="day"></div>
+        <div class="day"></div>
```

Comment:
```suggestion
      <div class="calendar calendar--start-day_thu calendar--month-length_30">
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
```

давай виправимо згідно BEM